lib/libzpool/uzfs_mgmt.c

```diff
@@ -496,7 +507,11 @@ uzfs_zvol_create_cb(const char *ds_name, void *arg)
 		if (zv->zv_target_host[0] == '\0') {
 			LOG_ERR("target IP address is empty for %s", ds_name);
 			uzfs_close_dataset(zv);
-			return (EINVAL);
+
+			/* Skip rebuild_clone volumes */
```

Comment:
Comment can be like "For internally created clones, target_host doesn't exists"
Shall we move this condition to the starting of the function where check for '@' is done? (or) a macro can also be made if this need to be checked at multiple places

Reply:
Done.